airflow/config_templates/config.yml

```diff
@@ -628,25 +628,25 @@
     - name: statsd_on
       description: |
         Enables sending metrics to StatsD.
-      version_added: ~
+      version_added: 2.0.0
```

Comment:
All metrics configuration options have been moved to a new section in Airflow 2.0, so they do not work on older versions.